src/OpenCV/ImgProc/MiscImgTransform.hs

```diff
@@ -422,3 +425,21 @@ watershed img markers =
                      , *$(Mat * markersPtr)
                      )
       }|]
+
+{- | Returns 0 if the pixels are not in the range, 255 otherwise. -}
+inRange ::
+     (ToScalar scalar)
+  => Mat ('S [w, h]) channels depth
+  -> scalar -- ^ Lower bound
+  -> scalar -- ^ Upper bound
+  -> CvExcept (Mat ('S [w, h]) ('S 1) ('S Word8))
```

Comment:
Will `fromBool` work on the `Mat`?

In any case, I think leaving it with `Word8` is OK, since it'll work with other functions like `matCopyTo`.